spec/std/openssl/cipher_spec.cr

```diff
@@ -44,4 +44,18 @@ describe OpenSSL::Cipher do
     s3.to_s.should eq(data)
     s3.to_slice.should eq(s4.to_slice)
   end
+
+  it "authenticated?" do
+    begin
+      cipher = OpenSSL::Cipher.new("aes-128-gcm")
+      cipher.authenticated?.should eq(true)
+    rescue ArgumentError
```

Comment:
Maybe add a check that its actually an expected ArgumentError by verifying the error message. It should be re-raised if it doesn't match.

Reply:
Could definitely do that, however if the `ArgumentError` is originating from anything other than the cipher not being available, it'd raise on the next invocation (line 58/59).